src/main/java/school/lemon/changerequest/java/Matrix.java

```diff
@@ -0,0 +1,126 @@
+package school.lemon.changerequest.java;
+
+/**
+ * Created by lera on 11.12.16.
+ */
+public class Matrix {
+
+    private int rows;
+    private int columns;
+    private double[][] matrix;
+
+
+    public Matrix(int rows, int columns) {
+        if ((rows < 1) || (columns < 1)) {
+            System.out.println("Cannot create a matrix smaller that 1x1");
+            System.exit(0);
+        } else {
+            this.rows = rows;
+            this.columns = columns;
+            this.matrix = new double[rows][columns];
+        }
+    }
+
+    public Matrix(double[][] matrix) {
+
+        for (int i = 1; i < matrix.length; i++) {
+            if (matrix[i].length != matrix[0].length)
+                throw new RuntimeException("Illegal vector dimensions"); //verification that all rows has the same columns number
+        }
+        this.rows = matrix.length;
+        this.columns = matrix[0].length;
+        this.matrix = new double[rows][columns];
+        for (int i = 0; i < rows; i++) {
+            for (int j = 0; j < columns; j++) {
+                this.matrix[i][j] = matrix[i][j];
+            }
+        }
+    }
+
+    public boolean checkSize(Matrix matrixB) {
+        if ((this.rows != matrixB.rows) || (this.columns != matrixB.columns))
+            throw new RuntimeException("Illegal vector dimensions.");
+        return true;
+
+    }
+
+    public Matrix addMatrix(Matrix matrixB) {
+
+        checkSize(matrixB);
+
+        Matrix matrixAplusB = new Matrix(rows, columns);
+        for (int i = 0; i < rows; i++) {
+            for (int j = 0; j < columns; j++) {
+                matrixAplusB.matrix[i][j] = matrix[i][j] + matrixB.matrix[i][j];
+            }
+        }
+        return matrixAplusB;
+    }
+
+    public Matrix subtractMatrix(Matrix matrixB) {
+
+        checkSize(matrixB);
+
+        Matrix matrixAsubtractB = new Matrix(rows, columns);
+        for (int i = 0; i < rows; i++) {
+            for (int j = 0; j < columns; j++) {
+                matrixAsubtractB.matrix[i][j] = matrix[i][j] - matrixB.matrix[i][j];
+            }
+        }
+        return matrixAsubtractB;
+    }
+
+
+    public Matrix multiplyMatrix(Matrix matrixB) {
+
+        if (this.columns != matrixB.rows) throw new RuntimeException("Illegal vector dimensions.");
+
+        Matrix multiplyMatrix = new Matrix(rows, columns);
+        double count = 0.;
+        for (int i = 0; i < rows; i++) {
+            for (int j = 0; j < matrixB.columns; j++) {
+                for (int index = 0; index < columns; index++) {
+                    count += matrix[i][index] * matrixB.matrix[index][j];
+                }
+                multiplyMatrix.matrix[i][j] = count;
+                count = 0.;
+            }
+        }
+        return multiplyMatrix;
+    }
+
+
+    public Matrix multiplyMatrixNumber(int number) {
```

Comment:
There is no need for different name of method.
I'd prefer all method names be:
* add
* sub
* div
* mul

Also there is no need to mention what exactly would be added\substracted\divided\multiplied, as you have argument with some Type, that will 100% say what would be done.